.vscode/extensions.json

```diff
@@ -0,0 +1,9 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
+  // Extension identifier format: ${publisher}.${name}. Example: vscode.csharp
+
+  // List of extensions which should be recommended for users of this workspace.
+  "recommendations": ["golang.go"],
+  // List of extensions recommended by VS Code that should not be recommended for users of this workspace.
+  "unwantedRecommendations": []
+}
```

Comment:
I think this file does not belong to this repository as it is related to your local development environment.

Reply:
Actually this file was intended. It just tells people using VS Code that the Go Lang extension is recommended. https://code.visualstudio.com/docs/editor/extension-gallery#_workspace-recommended-extensions

Reply:
I can obviously remove according to your preference, but I thought it as a welcome addition.

Reply:
I am not a maintainer of this repo, so my preference does not matter as much. I just thought that a developer's local development setup does not belong to a repo unless the repo itself dictates its development environment. If VIM, Emacs, Atom, and Sublime users start to add their project-specific configs then every repository will become a mess.